tools/src/main/java/org/apache/kafka/tools/StreamsResetter.java

```diff
@@ -281,7 +281,7 @@ private int maybeResetInputAndSeekToEndIntermediateTopicOffsets(final Map<Object
 
             if (!options.hasDryRun()) {
                 for (final TopicPartition p : partitions) {
-                    client.position(p);
+                    long pos = client.position(p);
```

Comment:
This is another hack for bypassing the error from SpotBugs about ignored return values.

Reply:
It is strange that we have to change this now.

Reply:
I agree. I still plan to dig into it more.

Reply:
Reverted. I'll look for another solution.